### DISCUSSION TITLE
Exit confirmation

### DISCUSSION THREAD

**Discussion by TravisGriggsProjects** [MitchellGriggsTAFE/TafeCalc24#5] [Ideas]
Currently when the user clicks the 'Exit button', it goes straight back to the main menu. Should we include a prompt that asks the user to confirm before exiting, in case they clicked it by accident?

**Comment by MitchellGriggsTAFE** [comment]
I don't believe that idea would be wise, as the user would then have to click through twice as many windows for each exit event, negatively impacting the user friendliness of the application. In any case, the user could simply click to re-enter the calculator they accidentally closed, as the exit button will return them to the main menu.
